### ISSUE TITLE
Any news in regards of Shelly Wave 2PM firmware update ? 

### ISSUE THREAD

**Issue by jfhautenauven** [QubinoHelp/Shelly_Wave_FW_OTA#1]
Hello,

A while ago, on this repo, the information has been shared that the latest firmware in date was causing trouble and issues and that this was related to Silicon Labs SDK. 

Since a few days, a new version of the SDK has been published that fixes some issues in regards with Zwave communication layer.

So here is the question : does that latest SDK version fix the issues for the Shelly Wave 2PM ? If yes, when can we expect a new firmware version for the Shelly Wave 2PM.

Thanks in advance

**Comment by QubinoHelp** [comment]
Latest news, the Silicon last confirm a bug and claim to fix it in the new SDK release that should be available next week. Once we have it we will build the firmware of the 2 PM on that and test it. Once we test it we will release the new firmware  :)

**Comment by jfhautenauven** [comment]
Cannot wait :) Keep me informed, I'm okay to beta test it if needed :)

**Comment by QubinoHelp** [comment]
This days we're testing the new SDK, first tests looks promising.

**Comment by pm-melo** [comment]
Hello @QubinoHelp! Any news regarding this matter?

Also, I noticed on [the website](https://support.shelly.cloud/en/support/solutions/articles/103000258471-shelly-qubino-wave-firmware-updates) that the ANZ firmware is more up-to-date than the remaining ones. Is this by design?

**Comment by jfhautenauven** [comment]
Hi @QubinoHelp, thanks for the update :) 

I've migrated since then to the Shelly WIFI Gen3 modules, sold my Zwave modules.

Thanks anyways for the good followup, as always :) 

Regards,

**Comment by marijsandy** [comment]
I have similar issues with QPDM-0A2P01EU (Wave Dimmer PRO 2 PM). I can't find any firmware in the https://github.com/QubinoHelp/Shelly_Wave_FW_OTA dir. Where can i find latest firmware for this device and has auto on/off issue when forwarding messages been resolved? Thanks

**Comment by QubinoHelp** [comment]
> I noticed that the file name for the new version (13) has a slighly differnt model code is that right or a typo?
> 
> [Wave_2PM_800_AU_20250129_1036_QNSW-002P14AU_[13.00]_9DD2F96C.gbl
> 
> 002P14AU "14" when the model is "16" so should be "002P16AU" ?
> 
> Cheers

This is likely due to a typing error in the filename during compilation.

**Comment by QubinoHelp** [comment]
> I have similar issues with QPDM-0A2P01EU (Wave Dimmer PRO 2 PM). I can't find any firmware in the https://github.com/QubinoHelp/Shelly_Wave_FW_OTA dir. Where can i find latest firmware for this device and has auto on/off issue when forwarding messages been resolved? Thanks

At the moment, there is no new firmware available for the Wave Dimmer Pro 2PM.

If you are referring to the random off issue that occurs when the device receives a routed command or acts as a relay for another command, this issue has not yet been resolved. The fix is expected to be included in the new SDK. We plan to start building firmware for other devices using the updated SDK end of the next month.

**Comment by Tschuuuls** [comment]
Have you basically deprecated the Wave series in favor of the new Gen 4 devices at this point?

**Comment by QubinoHelp** [comment]
We are still actively developing Z-Wave devices. At the moment, our focus is on Z-Wave LR for both EU and US regions. We have not yet released a newer firmware updates because we are waiting for Silicon Labs to release a stable SDK. 
I have added version 15.00 of the Wave 2PM firmware to the [GitHub repository] (https://github.com/QubinoHelp/Shelly_Wave_FW_OTA/tree/main/Wave_2PM/Temp).

**Comment by pm-melo** [comment]
Where does the 15.00 version comes from? I noticed that the latest version of the website is 12.23 ([ref](https://support.shelly.cloud/en/support/solutions/articles/103000258471-shelly-qubino-wave-firmware-updates#Shelly-Qubino-Wave-2PM))

**Comment by QubinoHelp** [comment]
The second number in the firmware updates represents bigger changes in the firmware, like updates to the SDK. The version we’re testing now has at least three newer SDK releases than version 12.23.

**Comment by miguelopezlopez** [comment]
Hello, first of all I would like to highlight this new version is much better in terms of stability. All disconnections I was suffering completly disapeared.
Are you planning to release this version for Shelly Wave 1PM?

**Comment by habibi-moose** [comment]
Are the firmware in the temp folder for wave 2PM v13 & v15 for EU testing firmware or stable/best/latest?

**Comment by QubinoHelp** [comment]
> Hello, first of all I would like to highlight this new version is much better in terms of stability. All disconnections I was suffering completly disapeared. Are you planning to release this version for Shelly Wave 1PM?

Yes, we're planning to release the Firmware updates for all the other devices, as explained few weeks ago we're developing the new firmware for all of the devices on the newer SDK which include also the LR capability.
As the firmware will be ready we will start to release the updates.

**Comment by QubinoHelp** [comment]
Thank you for the feedback.

You can downgrade to any version we didn't block the downgrade. In the Z-Wave JS start the update with the FW version you want. If you need a specific FW and you can't find it on the GitHub please let us know I will upload it to the repo.

**Comment by futorima-sketch** [comment]
Hi - is there a timeframe for the Wave 2PM firmware to be updated for the AU region? The dead node issue update would be really appreciated!

**Comment by axx-man** [comment]
Hi @QubinoHelp ,
I saw some updates in the past few weeks for a bunch of Wave devices, I am especially interested in an update for the Shelly Wave Pro 2PM device to support decoupled mode for inputs and outputs.
Any plans when an update for the Wave Pro 2PM devices is being published?

**Comment by lucas-dumartin** [comment]
> Shelly Wave Pro 2PM

I also have the same device with the same needs, I bought it expecting it would have the same functionality as the 2PM non pro I installed first.
Is there any ETA for the update on this device ?

**Comment by QubinoHelp** [comment]
@lucas-dumartin , @axx-man Wave Pro 2PM is uploaded, a PR was opened.

**Comment by axx-man** [comment]
@QubinoHelp Great thanks! I don't see any release notes regarding detached mode in Wave Pro 2PM, but I will see when update is applied. 😃 
My home assistant does not detect an update yet.. Shouldn't the update be detected already when clicking OTA update in Zwave JS UI?

**Comment by lucas-dumartin** [comment]
Thank you @QubinoHelp !
The update showed up today in my Zwave JS UI, and the detached mode is now present in the new software version @axx-man

**Comment by QubinoHelp** [comment]
> [@QubinoHelp](https://github.com/QubinoHelp) Great thanks! I don't see any release notes regarding detached mode in Wave Pro 2PM, but I will see when update is applied. 😃 My home assistant does not detect an update yet.. Shouldn't the update be detected already when clicking OTA update in Zwave JS UI?

The update is visible when the Pull request is merged in the main library and the data is refreshed also locally in the HA ;)

Closing issue

**Comment by axx-man** [comment]
Thanks for the hint. I restarted my whole home assistant system, now it found all the available zwave updates. Great to have the detached mode now!!
